### PR TITLE
New: Add `cac:Price` support to UBL

### LINKED ISSUE #955: Add `cac:Price` support to UBL
Implement the `cac:Price` element to represent unit pricing details, including `cbc:PriceAmount`.

### PR DESCRIPTION
**Pull request by alexmigf**
closes #955

### CODE REVIEW

File: ubl/Handlers/Invoice/InvoiceLineHandler.php [hunk 1]
```diff
@@ -152,4 +164,20 @@ public function handle( $data, $options = array() ) {
...
 
+	/**
+	 * Get the unit price of an item
+	 *
+	 * @param WC_Order_Item $item
+	 * @return int|float
+	 */
+	private function get_item_unit_price( $item ) {
+		if ( is_a( $item, 'WC_Order_Item_Product' ) ) {
+			return $item->get_subtotal() / $item->get_quantity();
```
Comment (MohamadNateqi) [inline ubl/Handlers/Invoice/InvoiceLineHandler.php:175]: Maybe you can use `get_item_subtotal()` method?
`$this->document->order->get_item_subtotal( $item, false, true );`

Comment (alexmigf) [reply ubl/Handlers/Invoice/InvoiceLineHandler.php:175]: Good catch!

Comment (MohamadNateqi) [reply ubl/Handlers/Invoice/InvoiceLineHandler.php:175]: Yes, it's the same. The only advantage might be using the provided source function or the applied filter. If those don't matter, the current code is fine, as it doesn't change over time.

Comment (alexmigf) [reply ubl/Handlers/Invoice/InvoiceLineHandler.php:175]: This seems good for now, and we can always tweak it later!

**Review by MohamadNateqi — APPROVED** [review]
LGTM!

**Pull request merged**